Code of conduct Section added in Readme.md (#1644)
conda-forge/conda-forge.github.io: README.md

We use https://hackmd.io/ for taking meeting minutes and will upload the resultant markdown file after the meeting has concluded.

There is a template provided in [`misc/DEV_MEETING_TEMPLATE.md`](https://github.com/conda-forge/conda-forge.github.io/tree/main/misc/DEV_MEETING_TEMPLATE.md) that you should use to create a new hackmd document.

## Code Of Conduct

> All are welcome to participate in conda-forge community (including the dev meetings) as long as the tenets of this Code of Conduct are adhered to:

Reviewer: ```suggestion
Everyone is welcome to participate in conda-forge community (including the dev meetings) as long as the tenets of this Code of Conduct are adhered to:
```

What do you think is more inclusive: "all are" or "everyone is"? Just pointing it out for you to decide :)

* Treat one another with dignity and respect.

> * Respect and treat others fairly, regardless of race ancestry, place of origin, colour, ethnic origin, citizenship, religion, gender, sexual orientation, age or disability.

Reviewer: ```suggestion
* Respect and treat others fairly, regardless of race, skin color, ethnicity, ancestry, place of origin, national origin, citizenship, religion, gender, sexual orientation, age, or disability.
```

Probably you should get the appropriate list from the reference source (numfocus) or follow some standard definition (e.g. in the US, there is a law about "protected characteristics" and most of these fall under that)

> * Behave professionally. Harassment and sexist, racist, or exclusionary jokes are condoned.

Reviewer: ```suggestion
* Behave professionally and patiently; respect everyone else's time and commitment. 
* Harassment and sexist, racist, or exclusionary jokes are unacceptable under any circumstance.
```

### Reporting Guidelines

> * The team at conda-forge takes reports of misconduct very seriously and is committed to preserving and maintaining the welcoming nature of our community.

Reviewer: ```suggestion
* We at conda-forge takes reports of misconduct very seriously and we are committed to preserving and maintaining the welcoming nature of our community.
```

Author: Thanks @ngam for the valuable feedback, will take a note of it

We use https://hackmd.io/ for taking meeting minutes and will upload the resultant markdown file after the meeting has concluded.

There is a template provided in [`misc/DEV_MEETING_TEMPLATE.md`](https://github.com/conda-forge/conda-forge.github.io/tree/main/misc/DEV_MEETING_TEMPLATE.md) that you should use to create a new hackmd document.

## Code Of Conduct
Everyone is welcome to participate in conda-forge community (including the dev meetings) as long as the tenets of this Code of Conduct are adhered to:
* Treat one another with dignity and respect.
* Respect and treat others fairly, regardless of race, skin color, ethnicity, ancestry, place of origin, national origin, citizenship, religion, gender, sexual orientation, age, or disability.
* Behave professionally and patiently; respect everyone else's time and commitment. 
* Harassment and sexist, racist, or exclusionary jokes are unacceptable under any circumstance.

### Reporting Guidelines
* We at conda-forge takes reports of misconduct very seriously and is committed to preserving and maintaining the welcoming nature of our community.

> * If you have experienced or witnessed behavior that violates the conda-forge Code of Conduct, please report this in a timely manner and complete the [form](https://numfocus.typeform.com/to/ynjGdT) to make a report. </br>

Author: > I don't think this break tag is necessary on line 50, since this is markdown and not HTML.

Will get it corrected, my bad 😅

> * If you have experienced or witnessed behavior that violates the conda-forge Code of Conduct, please report this in a timely manner and complete the [form](https://numfocus.typeform.com/to/ynjGdT) to make a report. </br>

Reviewer 2: Not a problem. That's what these reviews are for!
